From f06e150feb31826e103e18be410c562bc362a329 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Thu, 19 Apr 2012 23:56:26 +0200
Subject: [PATCH] transaction: remove a couple of asserts

We already asserted these facts in the previous loop.
(cherry picked from commit f1c2bdca422dba1bc5615f72662dee5ce69c147b)
---
 src/transaction.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/src/transaction.c b/src/transaction.c
index 39cfe54..41f7b82 100644
--- a/src/transaction.c
+++ b/src/transaction.c
@@ -611,9 +611,6 @@ static int transaction_apply(Transaction *tr, Manager *m, JobMode mode) {
                 /* We're fully installed. Now let's free data we don't
                  * need anymore. */
 
-                assert(!j->transaction_next);
-                assert(!j->transaction_prev);
-
                 /* Clean the job dependencies */
                 transaction_unlink_job(tr, j, false);
 
